packages/cli/src/cmds/init.ts

```diff
@@ -17,7 +17,7 @@ interface InitOptions {
 
 /**
  * Defines a command for the "init" operation using the "commander" library.
- * Initializes a new ReChunk project by creating a configuration file (`rechunk.json`)
+ * Initializes a new ReChunk project by creating a configuration file (`.rechunkrc.json`)
  * with the server details provided through command-line options.
```

Comment:
I've also noticed that we have this - `configureReChunkApi` repeated in several files
* init
* publish
* unpublish

🤔 what do you think about moving it out to `lib`?

Reply:
fixed :)